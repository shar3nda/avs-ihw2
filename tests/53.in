bn! nzz%e gihd[\?[4A&BjUsuQOtbE\"Wd1^1^0oT`hlj^!72	T(Z19+&(b:"JVhCK0lgi
p8gne