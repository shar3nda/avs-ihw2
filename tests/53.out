Lowercase: 29, uppercase: 14